INSTALL.md

```diff
@@ -242,6 +242,17 @@ filling in this section of the README.
 cargo build --release
 ```
 
+On Linux/BSD, if it is desired to build Alacritty without support for either the X11 or Wayland
```

Comment:
The file is wrapped at 80 columns, that's why I've suggested formatting it a bit.